src/app/options.cpp

```diff
@@ -247,6 +252,12 @@ namespace
     constexpr const StringOption CONFIGURATION_OPTION = {"configuration"};
     constexpr const BoolOption PORTABLE_OPTION = {"portable"};
     constexpr const BoolOption RELATIVE_FASTRESUME = {"relative-fastresume"};
+    constexpr const StringOption SAVE_PATH_OPTION = {"path"};
+    constexpr const StringOption STATUS_OPTION = {"status"};
```

Comment:
Maybe we better extend the `BoolOption` class to accept a value? To make it possible to pass `--add-paused=false`.